bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,307 @@
+| dateFormat                       | dateFormat                     | Date format                                                      | Number   | 0 = dd.mm.yyyy, 1 = mm.dd.yyyy, 2 = yyyy.mm.dd                                                       |
+| heatExchangerType                | heatExchangerType              | Heat exchanger type                                              | Number   | 0 = plastic, 1 = aluminium, 2 = enthalpy                                                             |
+| partyModeDuration                | partyModeDuration              | Party mode duration (in minutes)                                 | Number   | 5 - 180                                                                                              |
+| partyModeFanStage                | partyModeFanStage              | Party mode fan stage                                             | Number   | 0 - 4                                                                                                |
+| partyModeRemainingTime           | partyModeRemainingTime         | Party mode remaining time                                        | Number   | 0 - 180                                                                                              |
+| partyModeStatus                  | partyModeStatus                | Party mode status                                                | Switch   |                                                                                                      |
+| standbyModeDuration              | standbyModeDuration            | Standby mode duration (in minutes)                               | Number   | 5 - 180                                                                                              |
+| standbyModeFanStage              | standbyModeFanStage            | Standby mode fan stage                                           | Number   | 0 - 4                                                                                                |
+| standbyModeRemainingTime         | standbyModeRemainingTime       | Standby mode remaining time                                      | Number   | 0 - 180                                                                                              |
+| standbyModeStatus                | standbyModeStatus              | Standby mode status                                              | Switch   |                                                                                                      |
+| operatingMode                    | operatingMode                  | Operating mode (automatic/manual)                                | Switch   | 0 = automatic, 1 = manual                                                                            |

```

Comment:
I tried to stick with the variable names of the protocol specification. I do agree it would be self-explanatory. But then again a bit more inconsistent as well 😉

Reply:
That makes sense actually :)